templates/zerver/compose.html

```diff
@@ -78,6 +79,9 @@
                      id="attach_files" href="#" title="{{ _('Attach files') }}"></a>
                   <a class="message-control-button icon-vector-font"
                      href="#markdown-help" title="Formatting" data-toggle="modal"></a>
+                  <a class="preview_text" style="display:none;"
```

Comment:
I don't think the name `preview_text` is very intuitive.  You don't need to "preview" text; it's already text.